tf/env/local/secrets-recaptcha.tf

```diff
@@ -29,6 +34,11 @@ resource "kubernetes_secret" "recaptcha-v2-secrets" {
     "site_key"   = var.recaptcha_v2_site_key,
     "secret_key" = var.recaptcha_v2_secret
   }
+
+  depends_on = [
```

Comment:
Good idea to be explicit to avoid the race. I think if it works it would be even better to allow terraform to resolve the dependency itself.

I think this means on line 26 we should remove the raw `"adhoc-jobs"` string and replace it with a reference to a string from the kubernetes namespace resource (`kubernetes_namespace.adhoc-job-namespace.metadata[0].name`)

You can see an example of this in the namespaces.tf: https://github.com/wmde/wbaas-deploy/blob/9571a3c70a779dc2921dac9cb590cec52e100b34/tf/env/local/namespaces.tf#L10